blueflood-core/src/integration-test/java/com/rackspacecloud/blueflood/io/IntegrationTestBase.java

```diff
@@ -129,6 +129,26 @@ public void tearDown() throws Exception {
         // meh
     }
 
+    /**
+     * create a number of test locators of numTenants x numMetricNames combination
+     * @param numTenants
+     * @param numMetricNames
+     * @return list of locators for testing
+     */
+    protected List<Locator> generateTestLocators(int numTenants, int numMetricNames) {
```

Comment:
@shintasmith 
I moved it so I can use it for the excess enums test too.  I can create an IOIntegrationTestBasically if you'd like?